.travis.yml

```diff
@@ -6,5 +6,9 @@ node_js:
 sudo: required
 addons:
   chrome: stable 
-install: npm install -g npx && npx lerna bootstrap  
+cache:
+  directories:
+    - ~/.npm
+before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6; fi
```

Comment:
Maybe should document that npm 6 is required for local install as well. Might be possible to use the "prepare" script or something to actually test for this instead.  FFS `npm`…

Reply:
I plan to document the dev setup in another PR.